CMakeLists.txt

```diff
@@ -848,6 +848,7 @@ else()
         ADD_SUBDIRECTORY(os_stub/spdm_device_secret_lib_null)
         ADD_SUBDIRECTORY(unit_test/spdm_transport_test_lib)
         ADD_SUBDIRECTORY(unit_test/cmockalib)
+        ADD_SUBDIRECTORY(library/spdm_crypt_lib/fips)
```

Comment:
Does this need to be a separate library? Why not make it part of `spdm_crypt_lib`?

Reply:
Thanks. I have included the fips as the part of `spdm_crypt_lib`.